README.md

```diff
@@ -28,9 +28,26 @@ Solo es necesario tener instalado Node lts/dubnium (v10.16.0). Se recomienda usa
 #### Para desarrollo
 
 1. Ejecuta `npm install` para descargar todas las dependencias.
-2. Ejecuta `npm start` para correr el servidor de desarrollo. El mismo se encarga recargar las páginas en el navegador cuando guardas alguna modificación a los archvos del repositorio.
-3. Visita http://localhost:3000/health 
-4. Verificar que retorne cógigo 200 y un JSON { "status": "UP" }
+2. Exporta el puerto como variable de entorno, abre un terminal y ejecuta:
```

Comment:
The code loads all variables from a `.env` file, so this description should be updated with the way of doing things right now.
Could be useful to add something like a `.env.example` with a template of the currently used variables, so the developer just copy the contents to the `.env` and sets the values

Reply:
Did not remember that. 

It is good that I opened a PR for this, I actually wanted to make sure that I was not writing something that was not correct.

Reply:
I will make the changes and get back